flow/project.py

```diff
@@ -104,9 +104,18 @@ class _condition(object):
     current_arbitrary_tag = 0
 
     def __init__(self, condition, tag=None):
-        # Add tag to differentiate built-in conditions during graph detection.
+        '''Add tag to differentiate built-in conditions during graph detection.
+
+        Will raise a ``RuntimeError`` if tag is not set and does not have a code
```

Comment:
There's an implied "it" whose antecedent is unclear to me. Also specify that it's `__code__` and not `code`.
```suggestion
        Will raise a ``RuntimeError`` if tag is not set and the condition does not have a ``__code__``
```